src/components/index.js

```diff
@@ -3,5 +3,6 @@ import Home from "../pages/Home";
 import Footer from "./Footer";
 import Productlist from "../pages/Productlist";
 import Cart from "../pages/Cart";
```

Comment:
You will have to change these to named export in your components or you can use aliasing here to export without importing